src/client/sound/s_main.c

```diff
@@ -255,13 +312,22 @@ void S_Shutdown(void) {
 
 	S_Stop();
 
-	Mix_AllocateChannels(0);
+	if (s_env.sources[0]) {
+		alDeleteSources(MAX_CHANNELS, s_env.sources);
+		S_CheckALError();
+	}
 
 	S_ShutdownMusic();
 
 	S_ShutdownMedia();
+	
+	if (s_env.context) {
+		alcDestroyContext(s_env.context);
```

Comment:
I'm getting a 100% reproducible crash on this line. No idea.

![screen shot 2017-05-07 at 9 09 19 am](https://cloud.githubusercontent.com/assets/643118/25781216/df202d4e-3304-11e7-8e7d-6e0e2ae61cbe.png)